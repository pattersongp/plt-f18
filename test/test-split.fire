func int main = () => {
    str tmp = "i am just a test string";
    array[int,str] words; 
    words = split(tmp, " ");
    int i = len(words)-1;
    while (i >= 0) {
        sprint(words[i]);
        i = i-1;
    }
}
