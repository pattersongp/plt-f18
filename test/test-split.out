string
test
a
just
am
i
